js_modules/dagit/packages/core/src/assets/AssetDetails.tsx

```diff
@@ -23,9 +23,10 @@ import {AssetQuery, AssetQueryVariables} from './types/AssetQuery';
 interface Props {
   assetKey: AssetKey;
   asOf: string | null;
+  asSidebarSection?: boolean;
 }
 
-export const AssetDetails: React.FC<Props> = ({assetKey, asOf}) => {
+export const AssetDetails: React.FC<Props> = ({assetKey, asOf, asSidebarSection}) => {
```

Comment:
Can this stuff be done in a separate PR? I feel like major WS changes belong in a changeset separate from other product changes.